cloud/docker/docker.py

```diff
@@ -888,7 +908,7 @@ def get_start_params(self):
         optionals = {}
         for optional_param in ('devices', 'dns', 'volumes_from',
                 'restart_policy', 'restart_policy_retry', 'pid', 'extra_hosts',
-                'log_driver', 'cap_add', 'cap_drop', 'read_only', 'log_opt'):
+                'log_driver', 'cap_add', 'cap_drop', 'read_only', 'log_opt', 'ulimits'):
```

Comment:
Reading through the source, I don't think this is actually used! It adds `ulimits` to `optionals`, but then it never makes its way into `params`.